packages/dds/tree/src/feature-libraries/flex-tree/context.ts

```diff
@@ -175,20 +176,3 @@ export class Context implements FlexTreeHydratedContext, IDisposable {
 		return this.checkout.forest.events;
 	}
 }
-
-/**
- * A simple API for a Forest to interact with the tree.
- *
- * @param forest - the Forest
- * @param editor - an editor that makes changes to the forest.
- * @param nodeKeyManager - an object which handles node key generation and conversion.
- * @returns {@link FlexTreeContext} which is used to manage the cursors and anchors within the FlexTrees:
- * This is necessary for supporting using this tree across edits to the forest, and not leaking memory.
- */
-export function getTreeContext(
```

Comment:
This wrapper around the constructor has been removed as it no longer contains any of its own logic. Some of its docs have been preserved moving them to the constructor.